submissions/Jonah Onah/html/product.html

```diff
@@ -0,0 +1,59 @@
+<!DOCTYPE html>
```

Comment:
Well done! You are only meant to use HTML. Remove all styling of the images and any other styling used.

Reply:
Thank you Tolu. 

I do not have any styling for the images and any styling whatsoever in the project. Just plain html. 
The images looked like that because I added a 'width' attribute. I however have removed that also. 

thanks.